resources/src/commonMain/kotlin/dev/icerock/moko/resources/desc/StringDesc.kt

```diff
@@ -22,7 +22,7 @@ expect interface StringDesc {
 
     sealed class LocaleType {
         object System : LocaleType
-        class Custom(locale: String) : LocaleType
+        class Custom(languageTag: String) : LocaleType
```

Comment:
argument name change will broke public API. please rollback to `locale` name

Reply:
reverted to `locale` in 78ca4dfc656a52ed9c80c3510b4e946dc15a8730